solvedHW emil kacmzarski

EmilKaczmarski opened the pull request.

EmilKaczmarski committed:
solvedHW

ehyrol committed:
add LocationListViewController

EmilKaczmarski committed:
unvisible view resolved

EmilKaczmarski committed:
Merge branch 'master' of https://github.com/EmilKaczmarski/SwiftLivecoding

EmilKaczmarski committed:
-duplicated cities problem solved

-duplicated cities problem solved as hw

accuLeszek commented on mvcSample/mvcSample/Controller/ViewController.swift:
```diff
@@ -49,57 +52,115 @@ class ViewController: UIViewController {
+                    cityList.cities.forEach{ city in
+                        group.enter()
+                        WeatherRepository.getCurrentWeather(city.title) { weather, error in
+                            group.leave()
```
to wyjście z grupy powinno być po dodaniu informacji pogodowej do miasta ;)

EmilKaczmarski committed:
commit 7th

ui almost done, need to change saturation of colors and fix cell spacing, other things are ok

EmilKaczmarski committed:
Revert "commit 7th"

This reverts commit 2c8a3d1f071a3a77488a640295c71d4e250012cc.
